src/unity/python/turicreate/toolkits/style_transfer/style_transfer.py

```diff
@@ -179,8 +179,11 @@ def create(style_dataset, content_dataset, style_feature=None,
         max_iterations = 5000
         if verbose:
             print('Setting max_iterations to be {}'.format(max_iterations))
-    # data loader
 
+    if len(content_dataset) < batch_size:
+        raise _ToolkitError('Number of content images is less than the batch_size')
```

Comment:
The user doesn't explicitly choose the batch size - so this doesn't seem like a reasonable error to give. Since the batch size is not set by the user anyway, can't we just arbitrarily change it under the hood to avoid this error?

Reply:
Batch size is a parameter for ST/OD/IC now, so the user can and might have set this value.

However, I feel like we just replicate the content images to avoid this altogether.

Reply:
It appears to be an optional parameter. So we should think about the desired UX when:

* The user passes in an explicit batch size that is larger than `len(content_images)`
* The user does not pass in a batch size, but our default is larger than `len(content_images)`.

In both cases, could we simply set batch_size to len(content_images) (and/or, throw in the csae that the user explicitly provides a nonsensical batch size w.r.t. the number of content images)?